Gemfile.lock

```diff
@@ -176,6 +177,4 @@ DEPENDENCIES
   turbolinks
   uglifier (>= 1.3.0)
   web-console (~> 2.0)
-
```

Comment:
@Sakibs are you talking about the yahoofinance gem? All of this is auto generated code from the installed gems